src/test/java/org/pac4j/javalin/example/ExampleHttpActionAdapter.java

```diff
@@ -3,19 +3,17 @@
 import io.javalin.http.ForbiddenResponse;
 import io.javalin.http.UnauthorizedResponse;
 import org.pac4j.core.context.HttpConstants;
-import org.pac4j.javalin.DefaultHttpActionAdapter;
-import org.pac4j.javalin.Pac4jContext;
-
-public class ExampleHttpActionAdapter extends DefaultHttpActionAdapter {
+import org.pac4j.core.exception.http.HttpAction;
+import org.pac4j.javalin.JavalinHttpActionAdapter;
+import org.pac4j.javalin.JavalinWebContext;
 
+public class ExampleHttpActionAdapter extends JavalinHttpActionAdapter {
     @Override
-    public Void adapt(int code, Pac4jContext context) {
-        if (code == HttpConstants.UNAUTHORIZED) {
-            throw new UnauthorizedResponse();
-        } else if (code == HttpConstants.FORBIDDEN) {
-            throw new ForbiddenResponse();
-        } else {
-            return super.adapt(code, context);
+    public Void adapt(HttpAction action, JavalinWebContext context) {
+        switch(action.getCode()){
+            case HttpConstants.UNAUTHORIZED: throw new UnauthorizedResponse();
```

Comment:
This custom `HttpActionAdapter` does the same thing as its parent. Please try to return some custom message for the messages.